tests/dash/conftest.py

```diff
@@ -0,0 +1,110 @@
+import logging
+
+import pytest
+
+from ipaddress import ip_interface 
+from constants import *
+from dash_utils import render_template_to_host, apply_swssconfig_file
+
+
+logger = logging.getLogger(__name__)
+
+
+def pytest_addoption(parser):
+    """
+    Adds pytest options that are used by DASH tests
+    """
+
+    parser.addoption(
+        "--skip_config",
+        action="store_true",
+        help="Apply new configurations on DUT"
+    )
+
+    parser.addoption(
+        "--config_only",
+        action="store_true",
+        help="Apply new configurations on DUT"
+    )
+
+@pytest.fixture(scope="module")
+def config_only(request):
+    return request.config.getoption("--config_only")
+
+@pytest.fixture(scope="module")
+def skip_config(request):
+    return request.config.getoption("--skip_config")
+
+@pytest.fixture(scope="module")
+def config_facts(duthost):
+    return duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
+
+@pytest.fixture(scope="module")
+def minigraph_facts(duthosts, rand_one_dut_hostname, tbinfo):
+    """
+    Fixture to get minigraph facts
+
+    Args:
+        duthost: DUT host object
+
+    Returns:
+        Dictionary containing minigraph information
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+
+    return duthost.get_extended_minigraph_facts(tbinfo)
+
+def get_intf_from_ip(local_ip, config_facts):
+    for intf, config in config_facts["INTERFACE"].items():
+        intf_ip = ip_interface(config.keys()[0])
+        for ip in config:
+            intf_ip = ip_interface(ip)
+            if str(intf_ip.ip) == local_ip:
+                return intf, intf_ip
+
+
+@pytest.fixture(scope="module")
+def dash_config_info(duthost, config_facts, minigraph_facts):
+    dash_info = {
+        VM_VNI: 4321,
+        VNET1_VNI: 1000,
+        VNET2_VNI: 2000,
+        REMOTE_CA_IP: "20.2.2.2",
+        LOCAL_CA_IP: "11.1.1.1",
+        REMOTE_ENI_MAC: "F9:22:83:99:22:A2",
+        LOCAL_ENI_MAC: "F4:93:9F:EF:C4:7E",
+        REMOTE_CA_PREFIX: "20.2.2.0/24",
+    }
+    loopback_intf_ip = ip_interface(config_facts["LOOPBACK_INTERFACE"].values()[0].keys()[0])
```

Comment:
suggest to modify this line to something like this:
loopback_intf_ip = ip_interface(list(list(config_facts["LOOPBACK_INTERFACE"].values())[0].keys())[0])

It will allow to run tests using python3 environment from sonic-mgmt docker, without modification - we will see in python3 error: TypeError: 'dict_values' object is not subscriptable